docs/troubleshoot/cli/cli-use-curl-to-troubleshoot.md

```diff
@@ -0,0 +1,101 @@
+# Using cURL to troubleshoot Zowe CLI
+
+When a REST API call fails on Zowe CLI, you can use cURL to troubleshoot.
+
+Run a command with cURL and compare its output with what is returned using Zowe CLI. This can help pinpoint whether the problem lies with z/OSMF or Zowe CLI, depending on which command returns an API error.
+
+## Understanding cURL commands
+
+A cURL command using the `https` protocol follows the basic syntax `curl <URL>`.
+
+Add cURL options to establish communication between Zowe CLI and z/OSMF, as in the following example command:
+
+```
+curl --location --request <API method> "https://<host>:<port>/<API>" --header "X-CSRF-ZOSMF-HEADER;" --insecure --user "<ID>:<PASSWORD>"
+```
+**NOTE:** Some terminals may require single quotes rather than double quotes.
+
+
+### `--location`
+
+Use `--location` to allow the server to redirect to a different URL, if needed. 
+
+When a different location exists and `--location` is not included in the command, the server responds with a 3XX status code.
```

Comment:
Instead of "When a different location exists" would it be clearer to say "When the server attempts to redirect"?